.travis.yml

```diff
@@ -4,13 +4,25 @@ sudo: false
 matrix:
     include:
         - python: 2.7
-          env: {TOX_ENV: py27cov, COVERAGE: 1}
+          env: {TOX_ENV: py27-test, TOX_ARGS: --show-skipped --with-coverage, COVERAGE: 1}
+        - python: 3.3
+          env: {TOX_ENV: py33-test, TOX_ARGS: --show-skipped}
+        - python: 3.4
+          env: {TOX_ENV: py34-test, TOX_ARGS: --show-skipped}
+        - python: 3.5
+          env: {TOX_ENV: py35-test, TOX_ARGS: --show-skipped}
```

Comment:
Just in the interest of making the Travis build page easier to read, is there an easy way we can:

- Avoid repeating `--show-skipped`, since we apparently want to pass it to tox every time for normal test builds?
- Avoid `--with-coverage` by triggering it when `COVERAGE=1` so we don't have to repeat ourselves?